topics/command-info.md

```diff
@@ -0,0 +1,96 @@
+# Command information
+
+Starting from Redis 7.0 we include an additional map in `COMMAND`'s
+reply, which contains the following fields:
```

Comment:
i meant to move the entire documentation about the reply to COMMAND INFO, not just the part about the new map.
The page in command.md can just say that it's an array of responses of COMMAND INFO, and maybe give a quick example to demonstrate the nesting.

Reply:
i think it's better the way it is now:  none of the pages are too-full nor too-empty, feels like a good balance for the reader.

@itamarhaber WDYT?

Reply:
i think it's not logical, to describe part of the response in one page and another part in another page.
also, this has nothing to do with the command (i.e. there's no reason to document this part in INFO, and the other part in COMMAND, could be the other way around).

I'm ok with documenting the whole thing in COMMAND, and have INFO just refer to it..
but i was thinking it's cleaner to document the whole thing in INFO and have COMMAND refer to it.